### DISCUSSION TITLE
Will this repository remain a fork?

### DISCUSSION THREAD

**Discussion by BennettFourr** [vladmandic/sdnext#161] [Q&A] [answered]
Will this repository ever be switched from a fork to a separate repository?

**Comment by vladmandic** [comment, marked as answer]
undecided, there are pros-and-cons to both.
while it remains a fork, its easier to merge prs posted on a1111 main repo - and that is one of the main reason this exists, so it can move a bit quicker.

**Comment by BennettFourr** [comment]
Is the a1111 repo dead

**Reply by vladmandic** [reply]
probably not, just a1111 is taking a break and backlog of fixes is quite large.
the core issue is that a1111 is just one man - my goal is to have multiple maintainers (if there are volunteers) so it never relies on single person alone.
